spring-cloud-aws-messaging/src/test/java/org/springframework/cloud/aws/messaging/config/annotation/SqsConfigurationTest.java

```diff
@@ -119,6 +121,9 @@ void messageHandler_withFactoryConfiguration_shouldUseCustomValues()
 		assertThat(messageHandler.getCustomReturnValueHandlers().get(0)).isEqualTo(
 				ConfigurationWithCustomizedMessageHandler.CUSTOM_RETURN_VALUE_HANDLER);
 
+		Object sqsMessageDeletionPolicy = ReflectionTestUtils.getField(messageHandler,"defaultSqsMessageDeletionPolicy");
```

Comment:
there should be a space before `"defaultSqsMessageDeletionPolicy"`

Reply:
Done